index.js

```diff
@@ -85,6 +85,19 @@ Mailgen.prototype.generatePlaintext = function (params) {
     // Render the plaintext theme with ejs, injecting the data accordingly
     var output = ejs.render(this.cachedPlaintextTheme, ejsParams);
 
+    // Definition of the <br /> tag as a regex pattern
+    var breakTag = /(?:\<br\s*\/\>)/g;
```

Comment:
Now that I think of it, it would be nice to also support `<br>` tags which are valid in HTML as well. So let's add a `?` after the `\s*\/` in the regular expression. What do you think?